packages/component-library/stories/CivicCard.story.js

```diff
@@ -52,6 +171,35 @@ function DemoCardVisualization({ isLoading, data }) {
   );
 }
 
+function DemoCardVisualizationWithSelector({ isLoading, data }) {
+  const grpLabel = "Bus Line";
+  const radioLabels = ["14", "72", "20"];
+  const [busRoute, setBusRoute] = useState("14");
+  return (
+    !isLoading &&
+    data && (
+      <>
+        <RadioButtonGroup
+          grpLabel={grpLabel}
+          labels={radioLabels}
+          onChange={({ target }) => setBusRoute(target.value)}
+          value={busRoute}
```

Comment:
Thought I saw `row` here when you were running this locally?

Reply:
Row looked pretty ugly when I implemented it, so I removed it.